src/comment.rs

```diff
@@ -2,7 +2,7 @@
 
 use std::{borrow::Cow, iter};
 
-use itertools::{MultiPeek, multipeek};
+use itertools::{Itertools as _, MultiPeek, multipeek};
```

Comment:
Could you explain why you're brining `Itertools` into scope.

Reply:
It is used for the `Itertools::join` on line 1077.